BUILD.gn

```diff
@@ -165,9 +214,9 @@ copy("atom_js2c_copy") {
 
 action("atom_js2c") {
   deps = [
-    ":atom_browserify_content_script",
-    ":atom_browserify_isolated",
-    ":atom_browserify_sandbox",
+    ":atom_browserify_content_script_wrap",
+    ":atom_browserify_isolated_wrap",
+    ":atom_browserify_sandbox_wrap",
```

Comment:
i'd prefer we label the unwrapped targets as `_unwrapped` and have the ones we actually depend on be unsuffixed, but it's not a big deal